mailbox/api/src/main/java/org/apache/james/mailbox/model/MailboxMetaData.java

```diff
@@ -65,6 +64,10 @@ public static MailboxMetaData unselectableMailbox(MailboxPath path, MailboxId ma
         return new MailboxMetaData(path, mailboxId, delimiter, Children.CHILDREN_ALLOWED_BUT_UNKNOWN, Selectability.NONE);
     }
 
+    public static  final Comparator<MailboxMetaData> COMARATOR = Comparator
+        .<MailboxMetaData, Boolean>comparing(metadata -> metadata.getPath().isInbox())
```

Comment:
please provide evidences that the code you rewrite is well covered by tests with every commit